libafl/src/events/llmp.rs

```diff
@@ -689,8 +689,10 @@ where
     OT: ObserversTuple<I, S> + serde::de::DeserializeOwned,
     S: DeserializeOwned,
 {
-    #[cfg(target_os = "android")]
-    AshmemService::start().expect("Error starting Ashmem Service");
+    use crate::bolts::{os::unix_shmem_server::ServedShMemService, shmem::UnixShMemProvider};
+
+    let _service =
+        ServedShMemService::<UnixShMemProvider>::start().expect("Error starting Ashmem Service");
```

Comment:
@xpcmdshell yes, you'll need to spawn a shmem service like this. Didn't adapt the harnesses yet

Reply:
Also we may want to do this automatically in the broker, somehow, @s1341 what do you think?

Reply:
We can't do it in the broker. We need to do it before the `ShMemProvider` is created, which is before the launcher and broker are created.